Sources/SwiftGRPCNIOInteroperabilityTests/InteroperabilityTestCases.swift

```diff
@@ -17,6 +17,18 @@ import Foundation
 import SwiftGRPCNIO
 import NIOHTTP1
 
+/// Missing tests:
+/// - client_compressed_unary
+/// - server_compressed_unary
+/// - client_compressed_streaming
+/// - server_compressed_streaming
+/// - compute_engine_creds
+/// - jwt_token_creds
+/// - oauth2_auth_token
+/// - per_rpc_creds
+/// - google_default_credentials
+/// - compute_engine_channel_credentials
+
```

Comment:
Super-nit: Should there be a `///` on this line, too?

Reply:
No, it's is distinct from `EmptyUnary`. When we have a CLI, we'll have something to map test names (as listed in the spec, e.g. `empty_unary`) to test instances; that will be a better place for the list of missing tests to live.

Reply:
All right, thanks!